### PR TITLE
Disable selecting files for a review in the editor tools (bug 1197261)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 1197261](https://bugzilla.mozilla.org/show_bug.cgi?id=1197261) (again)

### CODE REVIEW

**Comment by magopian** [comment]
@wagnerand could you please try that out with a few different scenarios, and let me know if it does work as expected? Thanks!

File: apps/editors/templates/editors/review.html [hunk 1]
```diff
@@ -163,24 +163,20 @@ <h3 id="history">
...
-           data-value="{{ actions_minimal|join("|") }}"{% if allow_unchecking_files %} data-uncheckable="1"{% endif %}>
+           data-value="{{ actions_minimal|join("|") }}">
         <label for="id_addon_files"><strong>{{ form.addon_files.label }}</strong></label>
         <ul>
             {% for pk, label in form.fields.get('addon_files').choices %}
             <li>
               <label for="file-{{ pk }}"{% if pk in form.addon_files_disabled %} class="light"{% endif %}>
                 <input id="file-{{ pk }}" type="checkbox" value="{{ pk }}" name="addon_files"
-                       {% if pk in form.addon_files_disabled %}disabled=""{% endif %} />
+                       disabled="disabled" checked="checked" />
```
Comment (muffinresearch) [inline apps/editors/templates/editors/review.html]: Doesn't this just need to be `disabled` without any value as it's a boolean attribute?

File: apps/editors/tests/test_views.py [hunk 2]
```diff
@@ -2656,14 +2656,6 @@ def test_pending_to_public_unlisted_addon(self, mock_sign):
 
         assert mock_sign.called
 
-    def test_disabled_file(self):
```
Comment (muffinresearch) [inline apps/editors/tests/test_views.py]: What about a test to cover the new behaviour?

File: apps/editors/templates/editors/review.html (hunk 1 above)
Comment (magopian) [inline apps/editors/templates/editors/review.html]: The same for `checked` I believe, I was used to writing it the "old way", but i'll modify, thanks.

File: apps/editors/tests/test_views.py (hunk 2 above)
Comment (magopian) [inline apps/editors/tests/test_views.py]: Indeed!

**Comment by magopian** [comment]
Updated, what do you think @muffinresearch ?

@wagnerand whenever you have time to double check that, please let me know! (or I merge it and you test it on -dev?)

**Comment by muffinresearch** [comment]
@magopian lgtm

File: apps/editors/helpers.py [hunk 3]
```diff
@@ -799,7 +799,7 @@ class ReviewFiles(ReviewBase):
 
     def set_data(self, data):
         self.data = data
-        self.files = data.get('addon_files', None)
+        self.files = self.version.files.all()
```
Comment (wagnerand) [inline apps/editors/helpers.py]: This should be
```
self.files = self.version.files.all().filter(status=amo.STATUS_UNREVIEWED)
```

**Comment by wagnerand** [comment]
r+wc

Thank you!

File: apps/editors/helpers.py (hunk 3 above)
Comment (magopian) [inline apps/editors/helpers.py]: Actually, I removed this modification entirely, there was no reason to change anything here

**Pull request merged**